### ISSUE TITLE
Aztec: insert Media's ALT field, and make it editable.

### ISSUE THREAD

**Issue by diegoreymendez** [wordpress-mobile/WordPress-iOS#7860]
As a result of #7837, Media objects now have an `alt` field that should be inserted in posts when images are inserted.

Tasks:
- When an image is inserted into a post, we should also insert the media object's `alt` field in the `img` tag's `alt` attribute.
- The img tag's value should be loaded and editable at the image details edit screen (tap on image to edit its details), but it should NOT affect the media library value.  The media library value should never be affected by changes to specific posts.
- The alt value should also be the one that VoiceOver reads out.

Note 1: This issue only covers the new `Aztec` editor, and intentionally avoids diverting effort into the previous (hybrid) editor.

Note 2: Shall any work be necessary in the Aztec library, a separate issue should be opened in its repository.
